src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -35,12 +37,51 @@ public Settings(IConfigurationSource source)
             JaegerExporterAgentHost = source.GetString(ConfigurationKeys.JaegerExporterAgentHost) ?? "localhost";
             JaegerExporterAgentPort = source.GetInt32(ConfigurationKeys.JaegerExporterAgentPort) ?? 6831;
 
+            LoadTracerAtStartup = source.GetBool(ConfigurationKeys.LoadTracerAtStartup) ?? true;
+
+            ConsoleExporterEnabled = source.GetBool(ConfigurationKeys.ConsoleExporterEnabled) ?? true;
+
+            EnabledInstrumentations = Enum.GetValues(typeof(IntegrationIds)).Cast<IntegrationIds>().ToList();
```

Comment:
Is the Enum.GetValues exception readable? I mean does it points exactly what is wrong in this case?